src/pages/experiments/[experimentId]/plots-and-tables/embedding-categorical/index.jsx

```diff
@@ -38,7 +40,7 @@ const route = {
 const EmbeddingCategoricalIndex = ({ experimentId }) => {
   const dispatch = useDispatch();
   const config = useSelector((state) => state.componentConfig[plotUuid]?.config);
-  const cellSets = useSelector((state) => state.cellSets);
+  const cellSets = useSelector(getCellSets());
```

Comment:
could this part:
```
const hierarchy = cellSets.hierarchy.map((cellSet) => ({
      key: cellSet.key,
      children: cellSet.children?.length || 0,
    }));
```
perhaps be moved into a selector? or reuse the selector I mentioned making in other comments (perhaps having a default case where it gets what this is doing?) Whatever you think is cleaner